windows-pull-run.bat

```diff
@@ -0,0 +1,16 @@
+@echo off
+
+if not "%1"=="" (
+    if not "%1"=="-o" (
+           echo script usage: windows-pull-run.bat [-o]
+           echo -o: disables automatic opening of the browser
+           exit /b 1
+       )
+)
+docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:latest
+if not "%1"=="-o" start http://localhost:4000
+
+docker run -p 4000:3000 ghcr.io/badgerloop-software/chase-car-dashboard-image:latest
```

Comment:
What's the point of switching from port 4000 to 3000 or vice versa, why can't we just expose the port we're using?

Reply:
We have it this way so that automatically opening to port 4000 is easier. When we were testing we found that exposing port 4000 in the docker file was not working as expected it and it was slightly more difficult to have the new window pop up when some one runs the script.

Reply:
Fixed